Blockzilla.xcodeproj/project.pbxproj

```diff
@@ -14,10 +14,10 @@
 		0BBBED411DD3823C00F9C565 /* BaseTestCase.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0BBBED401DD3823C00F9C565 /* BaseTestCase.swift */; };
 		0BBBED431DD3B78300F9C565 /* SettingAppearanceTest.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0BBBED421DD3B78300F9C565 /* SettingAppearanceTest.swift */; };
 		0BC928CE1E366A6E004AC581 /* AsianLocaleTest.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0BC928CD1E366A6E004AC581 /* AsianLocaleTest.swift */; };
-		16074B6F21144206003B671F /* TrackingProtectionViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 16074B6E21144206003B671F /* TrackingProtectionViewController.swift */; };
```

Comment:
I know these are generated by Xcode, but can you uncommit them so as not to clutter the commit history? :)